packages/web/src/common/components/Table.tsx

```diff
@@ -84,80 +97,87 @@ const Table = <T,>({
 
   return (
     <TableWithCount>
-      {count && (
-        <Typography fs={16} lh={20}>
-          총 {count}개
-        </Typography>
-      )}
-      <TableInner height={height}>
-        <Header>
-          {table.getHeaderGroups().map(headerGroup => (
-            <HeaderRow key={headerGroup.id}>
-              {headerGroup.headers.map(header => {
-                if (header.isPlaceholder) {
-                  return null;
-                }
-                if (header.column.getCanSort()) {
+      <Count>
+        {count && (
+          <Typography fs={16} lh={20}>
+            총 {count}개
+          </Typography>
+        )}
+      </Count>
+      <TableInnerWrapper>
+        <TableInner height={height}>
+          <Header>
+            {table.getHeaderGroups().map(headerGroup => (
+              <HeaderRow key={headerGroup.id}>
+                {headerGroup.headers.map(header => {
+                  if (header.isPlaceholder) {
+                    return null;
+                  }
+                  if (header.column.getCanSort()) {
+                    return (
+                      <TableCell
+                        key={header.id}
+                        width={header.getSize()}
```

Comment:
header만 이상한 이유 여기였다! 요거 바꿔줄 수 있엉?

```suggestion
                        width={header.column.getSize()}
```